bigbluebutton-html5/imports/ui/components/whiteboard/component.jsx

```diff
@@ -17,6 +17,9 @@ import "@tldraw/tldraw/tldraw.css";
 import SlideCalcUtil from "/imports/utils/slideCalcUtils";
 import { HUNDRED_PERCENT } from "/imports/utils/slideCalcUtils";
 // eslint-disable-next-line import/no-extraneous-dependencies
+import Settings from "/imports/ui/services/settings";

```

Comment:
```suggestion
import Settings from '/imports/ui/services/settings';
```

I resolved the conflict incorrectly here.